src/YOUR_NAME_ex/ex05/bounded_sim.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Kevin Martin Lankut'
+__email__ = 'kela@nmbu.no'
+
+from walker_sim import Walker, Simulation
+import random
+
+
+class BoundedWalker(Walker):
+    def __init__(self, start, home, left_limit, right_limit):
+        super().__init__(start, home)
+        """
+        Initialise the walker
+
+        Arguments
+        ---------
+        start : int
+            The walker's initial position
+        home : int
+            The walk ends when the walker reaches home
+        left_limit : int
+            The left boundary of walker movement
+        right_limit : int
+            The right boundary  of walker movement
+        """
+        self.start = start
+        self.home = home
+        self.left_limit = left_limit
+        self.right_limit = right_limit
+
+    def bounded_move(self):
+        if self.start != self.left_limit and self.start != self.right_limit:
+            Walker.move(self)
```

Comment:
should  be `super().move()` super is always a proxy for the superclass. Logically its the same as what you have written, but if you decide to rename the walker class, you would have to rename all its references as well.